src/firewall/core/rich.py

```diff
@@ -871,50 +872,43 @@ def check(self):
         self._check_entry(self.source)
         self._check_entry(self.destination)
 
-        # service
         if isinstance(self.element, Rich_Service):
+            # service
```

Comment:
Just remove the comment entirely. It's clear from the object name this is a `Rich_Service`. The comment is not providing any value.

Reply:
dropped